packages/ui/src/bounty/components/BountyListItem/components/FundingDetails.tsx

```diff
@@ -20,7 +20,8 @@ export const FundingDetails = memo(({ fundingType, totalFunding, cherry }: Props
 
   if (!isFundingLimited(fundingType)) {
     const { target } = fundingType
-    const currentProgress = totalFunding.div(fundingType.target).toNumber()
+    // BN.js doesn't support decimals: https://github.com/indutny/bn.js/issues/213#issuecomment-475716946
+    const currentProgress = totalFunding.toNumber() / target.toNumber()
```

Comment:
That makes sense but generally we should avoid converting these BNs into numbers directly.
This would give enough precision:
```suggestion
    const currentProgress = totalFunding.muln(100).div(target).toNumber() / 100
```